lib/que/job.rb

```diff
@@ -77,7 +77,11 @@ def enqueue(*args)
         end
 
         if Que.mode == :sync && !t
-          run(*attrs[:args])
+          command_result = {
+            job_args: JSON.parse(JSON_MODULE.dump(attrs[:args]))
```

Comment:
Bit weird to use `JSON_MODULE` for generation and `JSON` for parsing, isn't it?

Also the opposite of `parse` is `generate`, so you'll probably want to use that.